simulation/plane.py

```diff
@@ -100,10 +98,14 @@ def __init__(
         self._f_lift = np.array([0.0, 0.0])
 
         # Sprite info
-        plane_pos = \
-            np.array(env_data["window_dimensions"]) * \
-            np.array(plane_data["starting_config"]["initial_position"]) // \
-            100
+        plane_pos = [
+            coord + 
+            random.uniform(
+                -plane_data["starting_config"]["position_px_deviation"], 
+                plane_data["starting_config"]["position_px_deviation"]
+            ) for coord in plane_data["starting_config"]["initial_position"]
+        ]
+
```

Comment:
a - dit is geen numpy array

b - kan dit niet in één keer? Het voelt wat omslachtig om het zo te doen. bijvoorbeeld zoiets:
```py
plane_pos = np.array([random.uniform(
    -plane_data["starting_config"]["position_px_deviation"], 
    plane_data["starting_config"]["position_px_deviation"],
    2
)]) + np.array(plane_data["starting_config"]["initial_position"])
```
Zo heb je geen loop nodig